RoboFile.php

```diff
@@ -130,8 +130,8 @@ public function createInstallZip()
             ->option('--include', 'www/runMigrations.php')
             ->option('--include', 'www/.htaccess')
             ->option('--include', 'www/media/consultations/1/consultation_thumb_micro_scholl.jpg')
-            ->option('--include', 'www/media/folders/misc/logo.png')
-            ->option('--include', 'www/media/folders/misc/epartool_logo.png')
+            ->option('--include', 'www/media/folders/misc/logo.svg')
+            ->option('--include', 'www/media/folders/misc/epartool_logo.svg')
```

Comment:
This should probably read just `logo.svg`, it's no longer logo of ePartool, but of DBJR itself.

So we really get rid of ePartool logo and use the same logo both in header and footer? Couldn't find more in the issue.

Reply:
@adamkudrna: Thank you for pointing that out. I don't know if I understand entirely which graphic refers to which place in the layout, but I try to describe how it should be:
- upper left corner (header): ePartool logo in white (because of coloured background) - that is the place where most installations will replace it with their own logo afterwards
- footer: 1 cell says something like "The tool for digital participation" and there it should show the ePartool logo in its non-white variant
- footer: 1 cell has the DBJR logo in it, and the DBJR logo has been refurbished a bit last autumn. Probably the "new logo" issue is only about that?

Reply:
I see. So it should either read `logo` and `stakeholder_logo` (or `backed_by_logo`, whatever suits best), but definitely there should **not** be `epartool` present as this is probably the first thing to get changed after installation of the tool. Is that correct?

Reply:
@adamkudrna no, this is sample data used in installation wizard. In this context `epartool_logo` is a very descriptive name.

Other stakeholders can remove this file and upload their own logo with filename reflecting the name of their organization.

Reply:
@DBJRdev 
I think this issue might be invalid. I'm not aware of the DBJR logo being used anywhere in the application at this time.

What @crishpeen did is that he changed the only logo in the system to be DBJR logo. According to https://github.com/DeutscherBundesjugendring/epartool/pull/39#discussion_r265929809 this was wrong since we want to keep the ePartool logo as is.

The only place where we could apply the DBJR logo is the third bullet in https://github.com/DeutscherBundesjugendring/epartool/pull/39#discussion_r265929809, but this footer entry does not exist.

Now we have two options:
1. Close this PR without merging.
2. Add the DBJR entry including logo into the footer. For this we would need the HTML to show in the footer and  info in which footer box it should appear (1-4 left to right). We are talking about this page https://mitwirkung.dbjr.de/mitmachen/admin/settings/footer in the reference installation.

Reply:
@DBJRdev how should we proceed?